src/oracle/PCVOracle.sol

```diff
@@ -28,6 +28,20 @@ contract PCVOracle is IPCVOracle, CoreRefV2 {
     /// know the USD value of PCV deployed in a given venue.
     mapping(address => address) public venueToOracle;
 
+    /// @notice track the venue's profit and losses
+    struct VenueData {
+        /// @notice the decimal normalized last recorded balance of PCV Deposit
+        int128 lastRecordedBalance;
+        /// @notice the decimal normalized last recorded profit of PCV Deposit
+        int128 lastRecordedProfit;
+    }
+
+    /// @notice venue information, balance and profit
+    mapping(address => VenueData) public venueRecord;
+
+    /// @notice cached total PCV amount
+    uint256 public totalRecordedPcv;
```

Comment:
i'd rename to `lastRecordedTotalPcv` and remove the getter `getTotalCachedPcv` because it should have the same value and the state variable is public

Reply:
done